man/homectl.xml

```diff
@@ -0,0 +1,807 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="homectl" conditional='ENABLE_HOMED'
+    xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>homectl</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>homectl</refentrytitle>
+    <manvolnum>1</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>homectl</refname>
+    <refpurpose>Create, remove, change or inspect home directories</refpurpose>
```

Comment:
I think a decision needs to be made whether to call this "homes" or "home directories". I think "home directories" sounds better and is will be much clearer to users. In particular, "home" is easily confused with `/home`. But homectl uses "homes", and I think this inconsistency is not great. (And yes, even if the underlying storage is a loopback device, what the user effectively gets and what they interact with, is a directory.)

Reply:
i prefer home directories too